putiosync/frontend.py

```diff
@@ -54,6 +66,16 @@ def parse_arguments():
         )
     )
     parser.add_argument(
+        "-f", "--filter",
+        default=None,
+        type=str,
+        help=(
+            "Filter for excluding or including specific files/folders from downloading."
+            "The filter is a regular expression (regex)"
+            "Example: putio-sync -f '/some/folder/*.avi' /path/to/Downloads"
```

Comment:
This is pretty minor, but I think this will look strange when output as the string concatenation will not put in any space/newline between the chunks.